src/api/exporters/ezsLocals.js

```diff
@@ -49,13 +57,13 @@ export function linkDataset(data, feed) {
         data.dataset = uri;
         data['@context'].dataset = {
             '@id': scheme,
-//             '@type': 'https://www.w3.org/TR/xmlschema-2/#anyURI',
+            //             '@type': 'https://www.w3.org/TR/xmlschema-2/#anyURI',
         };
     }
     feed.send(data);
 }
 
-export function JSONLDObject(data, feed) {
+export async function JSONLDObject(data, feed) {
```

Comment:
try and refactor this function without mutating a variable that has not be declared in the current scope.